library/src/interactor_impl.cxx

```diff
@@ -558,16 +498,97 @@ class interactor_impl::internals
     this->Window.render();
   }
 
+  struct tokenize_exception : public exception
+  {
+    explicit tokenize_exception(const std::string& what = "")
+      : exception(what){}
+  };
+
+  static std::vector<std::string> Tokenize(std::string_view str)
```

Comment:
```suggestion
  static std::vector<std::string> Tokenize(const std::string_view& str)
```

Reply:
nope, thats a string_view

Reply:
and? I agree the copy is cheaper than a string but still

Reply:
Ok I just checked online and it seems like while const-ref it is possible, it looks like common practice is to just copy it.